biz.aQute.resolve/src/biz/aQute/resolve/Bndrun.java

```diff
@@ -96,7 +98,7 @@ public String resolve(boolean failOnChanges, boolean writeOnChanges) throws Exce
 				if (!projectResolver.isOk()) {
 					return runbundlesFormatter.convert(Collections.<VersionedClause> emptyList());
 				}
-				Set<Resource> resources = resolution.keySet();
+				Collection<Resource> resources = getResources(resolution);
```

Comment:
Since the point of this PR is that the resources be ordered, this should be a `List<Resource>`, no?